CONTRIBUTING.md

```diff
@@ -687,6 +674,12 @@ This comment is required for the main patch in a sequence. See the following exa
 Signed-off-by: <contributor@foo.com>
 ```
 
+<!---
+  TODO:
+  The 'character limit' below is actually 75. This should be stated. 
```

Comment:
Agreed. The value is specified here by the way:

- https://github.com/kata-containers/tests/blob/690110f70cce450dc7376f2b69d010fd06f8f867/cmd/checkcommits/checkcommits.go#L58

But we may wish to set the value explicitly in the CI static check script here:

- https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh#L236..L240

.. by specifying the extra options `--subject-length 75 --body-length 72`.

That would make it easier for this doc to reference if required.

Reply:
I addressed both of these in new section "Pull request format."